### DISCUSSION TITLE
Install with jupyter?

### DISCUSSION THREAD

**Discussion by wwbrannon** [pyannote/pyannote-audio#1010] [General]
I'd like to use pyannote.audio in a jupyter notebook with conda. Do you know how to make this work? I've tried many configurations (the one in your Github README, installing via `pip install pyannote.audio`, jupyter in the same environment or another one) and can't get any of them to work. Errors are varied and difficult to diagnose: segfaults on importing pyannote.audio, but only from jupyter, link errors, and others.

I can go into more detail on any of these, but don't want to clutter up this discussion in case there's just a recipe to follow. Anything you can recommend?

Thanks

**Comment by hbredin** [comment]
`pip install pyannote.audio` will install the (soon-to-be-deprecated) 1.x branch of pyannote.audio.

Please install from the `develop` branch as stated in the README.

**Reply by remic33** [reply]
Is it already the case? There is no mention of changes from that in the readme.

**Comment by SpaceDA** [comment]
I just successfully installed it in a jupyter notebook in pycharm. Ran into some issues with dependencies (related to M1 I think) but ultimately was able to get it working on my M1 MacBook.
